packages/devops/scripts/ci/utils.sh

```diff
@@ -0,0 +1,22 @@
+ANSI_COLOR_GREEN="\e[32m"
+ANSI_COLOR_RED="\e[31m"
+ANSI_COLOR_RESET="\e[0m"
+ANSI_COLOR_YELLOW="\e[33m"
+
+function log_with_color() {
+  text=$1
+  color=$2
+  echo -e "$2$1$ANSI_COLOR_RESET"
+}
```

Comment:
Oooh this is very handy 😄

Reply:
Yeah color adds a new dimension to logs 🌈